[2025-02-14T23:05:58.141+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T23:05:58.172+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T23:05:21.937919+00:00 [queued]>
[2025-02-14T23:05:58.196+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T23:05:21.937919+00:00 [queued]>
[2025-02-14T23:05:58.197+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-14T23:05:58.225+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-02-14 23:05:21.937919+00:00
[2025-02-14T23:05:58.233+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1244) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T23:05:58.235+0000] {standard_task_runner.py:72} INFO - Started process 1245 to run task
[2025-02-14T23:05:58.237+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-02-14T23:05:21.937919+00:00', '--job-id', '352', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpmodbass4']
[2025-02-14T23:05:58.239+0000] {standard_task_runner.py:105} INFO - Job 352: Subtask assign_permissions_dag
[2025-02-14T23:05:58.328+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T23:05:21.937919+00:00 [running]> on host 71404c6b5320
[2025-02-14T23:05:58.507+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T23:05:21.937919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T23:05:21.937919+00:00'
[2025-02-14T23:05:58.509+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T23:06:01.389+0000] {dag.py:219} INFO - Assigned permissions to teste for DAG hello_world_dag
[2025-02-14T23:06:04.889+0000] {dag.py:219} INFO - Assigned permissions to teste for DAG hello_world_dag
[2025-02-14T23:06:04.890+0000] {logging_mixin.py:190} INFO - current_owner forhello_world_dag = teste
[2025-02-14T23:06:04.890+0000] {dag.py:201} INFO - Permission for DAG hello_world_dag is already correctly assigned to teste
[2025-02-14T23:06:04.890+0000] {logging_mixin.py:190} INFO - current_owner formanage_***_roles = data-team
[2025-02-14T23:06:04.890+0000] {dag.py:201} INFO - Permission for DAG manage_***_roles is already correctly assigned to data-team
[2025-02-14T23:06:04.891+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T23:06:04.897+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T23:06:04.897+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-02-14T23:05:21.937919+00:00, execution_date=20250214T230521, start_date=20250214T230558, end_date=20250214T230604
[2025-02-14T23:06:04.948+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T23:06:04.988+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14T23:06:04.990+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
